### PR TITLE
Fix incorrect or unneeded dependencies.

### PR DESCRIPTION
**Pull request by tech4him1**
**- Summary**

There were quite a few dependencies that were either in an incorrect location in the `package.json`, or are no longer used. I know we might end up changing this entirely, later, but I thought it would be good to at least have it correct for now.

**- Test plan**

WIP: Run `yarn` and test build.

**- Description for the changelog**

Fix incorrect or remove unneeded dependencies.

**- A picture of a cute animal (not mandatory but encouraged)**
[vita-vilcina-274033](https://unsplash.com/photos/jRF4UDDuzJc?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,11 +78,11 @@
     "exports-loader": "^0.6.4",
     "extract-text-webpack-plugin": "^2.1.2",
     "file-loader": "^0.11.2",
-    "gotrue-js": "^0.9.3",
```
Comment (tech4him1) [inline package.json]: To be consistent with what we have right now, I moved `gotrue-js` into the regular dependencies.

File: package.json [hunk 2]
```diff
@@ -102,29 +105,25 @@
...
   "dependencies": {
-    "@kadira/storybook": "^1.36.0",
-    "autoprefixer": "^6.3.3",
-    "babel-plugin-transform-builtin-extend": "^1.1.0",
     "classnames": "^2.2.5",
     "create-react-class": "^15.6.0",
     "dateformat": "^1.0.12",
     "deep-equal": "^1.0.1",
     "fuzzy": "^0.1.1",
+    "gotrue-js": "^0.9.3",
```
Comment (tech4him1) [inline package.json]: I moved `gotrue-js` into the regular dependencies since that is consistent with what we have right now.

**Comment by tech4him1** [comment]
@erquhart I moved `gotrue-js` into the regular dependencies since that is consistent with what we have right now, but I can move it back if you want.

File: package.json [hunk 3]
```diff
@@ -65,21 +67,23 @@
...
     "eslint-import-resolver-webpack": "^0.8.3",
     "exports-loader": "^0.6.4",
     "extract-text-webpack-plugin": "^2.1.2",
     "file-loader": "^0.11.2",
-    "gotrue-js": "^0.9.3",
     "identity-obj-proxy": "^3.0.0",
     "imports-loader": "^0.7.1",
     "jest": "^20.0.4",
     "jest-cli": "^20.0.4",
+    "json-loader": "^0.5.4",
```
Comment (erquhart) [inline package.json]: Looks like this isn't being used at all, no?

Comment (tech4him1) [reply package.json]: Done.

**Pull request merged**